pyhik/hikvision.py

```diff
@@ -285,16 +288,32 @@ def get_device_info(self):
 
         try:
             response = self.hik_request.get(url)
+            if response.status_code == requests.codes.unauthorized:
+                _LOGGING.debug('Basic authentication failed. Using digest.')
+                self.hik_request.auth = HTTPDigestAuth(self.usr, self.pwd)
+                response = self.hik_request.get(url)
+
             if response.status_code == 404:
```

Comment:
Can you change this to use the ```requests.codes``` also?  It's a good idea and makes things cleaner.